libs/algorithms/tests/unit/container_algorithms/transform_range_binary.cpp

```diff
@@ -20,6 +21,43 @@
 #include "test_utils.hpp"
 
 ///////////////////////////////////////////////////////////////////////////////
+template <typename IteratorTag>
```

Comment:
The newly added tests are never executed. Could you please check that they are invoked somewhere?

Reply:
Is it like I should call them without an exec-policy? Will they be invoked then?